### DISCUSSION TITLE
Set focus of PasswordBox

### DISCUSSION THREAD

**Discussion by DFFspace** [microsoft/WindowsAppSDK#2803] [Q&A] [answered]
I was wondering if there is any sort of way to make the PasswordBox focussed (always highlighted) on the main page.

At the moment I have to click in the box to make it Focus and when switching to settings and than back to the main page it lost focus.

**Comment by AndrewKeepCoding** [comment, marked as answer]
You can use the ``Loaded``  event and call the ``Focus`` method there.

```csharp
public sealed partial class MainPage : Page
{
    public MainPage()
    {
        this.InitializeComponent();
        this.Loaded += MainPage_Loaded;
    }

    private void MainPage_Loaded(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
    {
        this.MyPassswordBox.Focus(Microsoft.UI.Xaml.FocusState.Programmatic);
    }
}
```
